sdks/js/packages/spark-sdk/src/signer/signer.ts

```diff
@@ -87,16 +87,21 @@ type DerivedHDKey = {
   publicKey: Uint8Array;
 };
 
+type KeyPair = {
+  privateKey: Uint8Array;
+  publicKey: Uint8Array;
+};
+
 interface HDKeyGenerator {
   deriveHDKeysFromSeed(
```

Comment:
My only comment here is that since this is no longer just an HDKeyGenrerator perhaps we should rename this to something more precise.

Reply:
Addressed!